oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -154,8 +154,8 @@ protected static List<NetworkInterface> getNetworkInterfaces(boolean includeLoca
         List<NetworkInterface> interfaces = getAllNetworkInterfaces();
 
         return includeLocalInterfaces ? interfaces
-                : getAllNetworkInterfaces().stream().filter(networkInterface1 -> !isLocalInterface(networkInterface1))
-                        .collect(Collectors.toList());
+                : getAllNetworkInterfaces().stream().parallel()
+                        .filter(networkInterface1 -> !isLocalInterface(networkInterface1)).collect(Collectors.toList());
```

Comment:
Huh, that `networkInterface1` snuck in at some point, probably due to a refactoring. I like shorter names in lambdas, though, why not [`ni`](https://www.youtube.com/watch?v=zIV4poUZAQo)?

Reply:
hmmm how about just `.filter(Predicate.not(AbstractNetworkIF::isLocalInterface))`

Reply:
nevermind it seems has compilation errors
```
Error:  /home/runner/work/oshi/oshi/oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java:[159,42] cannot find symbol

```